app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,10 @@
     <meta name="distribution" content="GLOBAL" />
     <meta name="classification" content="transformative works" />
     <meta name="author" content="Organization for Transformative Works" />
-    <% if @user && @user.try(:preference).try(:minimize_search_engines?) || @work.is_a?(Work) && @work.users.all? {|u| u.try(:preference).try(:minimize_search_engines?)} %>
+    <% if @user && @user.try(:preference).try(:minimize_search_engines?) ||
```

Comment:
The code looks good! It's getting a bit long, though. Maybe we could put the logic in a helper method in application_helper.rb?